pkg/logentry/stages/metrics.go

```diff
@@ -58,6 +62,24 @@ func validateMetricsConfig(cfg MetricsConfig) error {
 			config.MetricType != MetricTypeHistogram {
 			return errors.Errorf(ErrMetricsStageInvalidType, config.MetricType)
 		}
+
+		// Set the idle duration for metrics
+		if config.IdleDuration != nil {
```

Comment:
don't we support already time.Duration config ?

Reply:
not in any pipeline stages?

Reply:
yes may be but then why not make it global ?